channeldb/invoices.go

```diff
@@ -2225,6 +2231,83 @@ func (d *DB) cancelHTLCs(invoices kvdb.RwBucket, invoiceNum []byte,
 	return invoice, nil
 }
 
+// settleHodlInvoice marks a hodl invoice as settled.
+//
+// NOTE: Currently it is not possible to have HODL AMP invoices.
+func (d *DB) settleHodlInvoice(invoices, settleIndex kvdb.RwBucket,
```

Comment:
It looks like `settleHodlInvoice` and `cancelInvoice` don't have test coverage, not sure if needed?

Reply:
They are not called directly but they are both part of multiple flows tested in [`invoices/invoiceregistry_test.go`](https://github.com/lightningnetwork/lnd/blob/master/invoices/invoiceregistry_test.go)


[`settleHodlInvoice`](https://github.com/lightningnetwork/lnd/blob/master/invoices/invoiceregistry_test.go#L340)
[`cancelInvoice`](https://github.com/lightningnetwork/lnd/blob/master/invoices/invoiceregistry_test.go#L326)

Reply:
Ah, thank you, I think I confused something. A working command to see the coverage is `go test -coverprofile=coverage.out -cover -coverpkg=./... ./invoices`. In this case one needs to know in which package it's tested beforehand, so it's better to work with `make unit-cover` for these checks.